### DISCUSSION TITLE
Multilabel_Mode

### DISCUSSION THREAD

**Discussion by xiaoyanLi629** [qubvel/segmentation_models.pytorch#479] [Q&A]
Hello,

I have a small question about the loss.

I want to do the multi_label classification. I followed the cars segmentation example and tried to change the code to work for my project. This is the loss function I want to use.

`segmentation_models_pytorch.losses.constants.MULTILABEL_MODE: str= 'multilabel'
`
I my code:
`loss = segmentation_models_pytorch.utils.losses.constants.MULTILABEL_MODE`

I also tried:
`loss = segmentation_models_pytorch.utils.losses.constants.MULTILABEL_MODE: str = "multilabel"`

However, it doesn't work. Can someone tell how to do this multilabel_mode?

Thank you!
Xiaoyan

**Comment by romanvelichkin** [comment]
According to docs: https://smp.readthedocs.io/en/latest/losses.html#segmentation_models_pytorch.losses.DiceLoss
you have to create loss class and inside it you set mode variable as ‘binary’, ‘multiclass’ or ‘multilabel’.

**Reply by xiaoyanLi629** [reply]
Yes. Thank you for letting me know! I already fixed tis problem a few years ago. I even didn't remember I have this question here.
